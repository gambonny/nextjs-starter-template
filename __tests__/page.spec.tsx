import { render } from '@testing-library/react'
import '@testing-library/jest-dom'

import Page from '../app/page'

describe('page', () => {
  it('renders the main page', () => {
    const { container } = render(<Page />)

    expect(container).toMatchSnapshot()
  })
})
