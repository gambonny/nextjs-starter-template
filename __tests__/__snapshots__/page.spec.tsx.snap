// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`page renders the main page 1`] = `
<div>
  <h1
    class="font-bold text-6xl text-center"
  >
    Hello world
  </h1>
</div>
`;
